src/components/Player/index.js

```diff
@@ -83,6 +92,11 @@ class Player extends React.Component {
           </ButtonInline>
         </div>
         <div className="player-content-action">
+          <ButtonInline onClick={() => onSetShuffleMode(shuffleMode)}>
```

Comment:
Since you only invert the shuffleMode boolean in the reducer, you don't need to pass the shuffleMode here. You could change it to `<ButtonInline onClick={onSetShuffleMode}>`. (1)